websites/P/Plex/presence.ts

```diff
@@ -136,25 +136,28 @@ function getTranslation(stringName: string): string {
 let isUploading = false;
 
 const uploadedImages: Record<string, string> = {};
-async function uploadImage(url: string): Promise<string> {
+async function uploadImage(urltoupload: string): Promise<string> {
 	if (isUploading) return "plex";
 
-	if (uploadedImages[url]) return uploadedImages[url];
+	if (uploadedImages[urltoupload]) return uploadedImages[urltoupload];
 	isUploading = true;
 
-	const file = await fetch(url).then(x => x.arrayBuffer()),
-		outputUrl = await fetch("https://bashupload.com", {
-			method: "POST",
-			body: file,
-		})
-			.then(x => x.text())
-			.then(x => x.match(/https(.*)/)?.[0]);
+	const file = await fetch(urltoupload).then(x => x.blob()), // Fetch the file as a Blob object
```

Comment:
These comments are redundant, and just repeat what the code is doing.
```suggestion
	const file = await fetch(urltoupload).then(x => x.blob()),
```